internal/manifests/ingester/ingester.go

```diff
@@ -0,0 +1,31 @@
+package ingester
+
+import (
+	"github.com/os-observability/tempo-operator/api/v1alpha1"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func BuildIngester(tempo v1alpha1.Microservices) []client.Object {
```

Comment:
Should we add some comments to exposed methods? it might be obvious sometimes. But when people jump into new projects, its nice to have a bit more context.